internal/services/instances.go

```diff
@@ -87,6 +93,29 @@ func (s *ServiceClient) Get(ctx context.Context, namespace, name string) (*model
 	return &service, nil
 }
 
+// GetInternalRoutes returns the internal routes of the service, finding them from the kubernetes services of the Helm release
+func (s *ServiceClient) GetInternalRoutes(ctx context.Context, namespace, name string) ([]string, error) {
+	servicesList, err := s.kubeClient.Kubectl.CoreV1().Services(namespace).List(ctx, metav1.ListOptions{
+		LabelSelector: "app.kubernetes.io/instance=" + names.ServiceReleaseName(name),
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "fetching the services")
+	}
+
+	internalRoutes := []string{}
+	for _, s := range servicesList.Items {
+		route := fmt.Sprintf("%s.%s.svc.cluster.local", s.Name, s.Namespace)
+		for _, port := range s.Spec.Ports {
+			if port.Port != 80 {
+				route += fmt.Sprintf(":%d", port.Port)
```

Comment:
So a service `S` with multiple ports `A`, `B`, etc. will look like `S:A:B:...`, right ? Intention ?
Would making a separate entry per port make more sense ?

Reply:
Hu, I didn't thought about a service with multiple ports. No, it was not intentional, it would be nicer to show them separately!

Thanks for the gotcha!